### DISCUSSION TITLE
One way to disguise traffic

### DISCUSSION THREAD

**Discussion by Ns2Kracy** [anacrolix/torrent#986] [Q&A] [answered]
I wrote a feature to share using webtorrent, but it seems that some ISPs are detecting information about the torrent and blocking that traffic making it unusable, is there any way to disguise the traffic sent?

**Comment by andry81** [comment]
try this: https://github.com/ValdikSS/GoodbyeDPI

**Reply by Ns2Kracy** [reply]
Thanks!

**Reply by anacrolix** [reply, marked as answer]
Also ensure MSE is enabled, use the crypto select and provides options in the client. 

Disabling DHT and only using https trackers might help too. Really a whole-network adjustment like the DPI suggested above, or a VPN are the best options.
